Updated cfg file for beampixel application for 2018 run

dinardo opened the pull request:
I changed from eras.Run2_2017 to eras.Run2_2018
and from PixelTracks_2017_cff to PixelTracks_cff
- Mauro.

Commit:
Updated cfg file for beampixel application for 2018 run

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
@dinardo, CMSSW_10_1_X branch is closed for direct updates. cms-bot is going to move this PR to master branch.
In future, please use cmssw master branch to submit your changes.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-22782/4149

cmsbuild commented:
A new Pull Request was created by  @dinardo (Mauro) for master.

It involves the following packages:

DQM/Integration


@vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please review it and eventually sign? Thanks.
@makortel, @threus, @batinkov, @thomreis this is something you requested to watch as well.
@davidlange6, @slava77, @fabiocos you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

Updated tracking region for pixel tracks

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-22782/4157

cmsbuild commented:
Pull request #22782 was updated. @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please check and sign again.

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

Adjourned process.pixelTracksTrackingRegions.RegionPSet.originHalfLength to more approriate value.

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-22782/4166

cmsbuild commented:
Pull request #22782 was updated. @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please check and sign again.

dinardo commented:
Error
